UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.FunctionalTests/Helper/TestConfiguration.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace UKHO.ExternalNotificationService.API.FunctionalTests.Helper
+{
+    public class TestConfiguration
+    {
+        protected IConfigurationRoot ConfigurationRoot;
+        public string ensApiBaseUrl;
+        public string payloadFolder;
+        public string payloadFileName;
+
+        public TestConfiguration()
+        {
+            ConfigurationRoot = new ConfigurationBuilder()
+                                .AddJsonFile("appsettings.json", false)
+                                .Build();
+
+            ensApiBaseUrl = ConfigurationRoot.GetSection("EnsApiUrl").Value;
+            payloadFolder = ConfigurationRoot.GetSection("PayloadFolder").Value;
+            payloadFileName = ConfigurationRoot.GetSection("PayloadFileName").Value;
+            
```

Comment:
change it to:
 EnsApiBaseUrl = ConfigurationRoot.GetSection("EnsApiUrl").Value;
PayloadFolder = ConfigurationRoot.GetSection("PayloadFolder").Value;
PayloadFileName = ConfigurationRoot.GetSection("PayloadFileName").Value;

Reply:
Done required modification.